lib/src/layer/polyline_layer.dart

```diff
@@ -10,7 +10,7 @@ class Polyline {
   final Key? key;
   final List<LatLng> points;
   final List<Offset> offsets = [];
-  final double strokeWidth;
+  double strokeWidth;
```

Comment:
This doesn't need to be mutable.

```suggestion
  final double strokeWidth;
```